CMakeLists.txt

```diff
@@ -206,7 +208,13 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 
 else()
     # Custom build flags.
-    set(QUIC_COMMON_FLAGS "-DQUIC_PLATFORM_LINUX -fms-extensions -fPIC -pthread -Wl,--no-as-needed -ldl")
+
+    set(QUIC_COMMON_FLAGS "-fms-extensions -fPIC -pthread -g -O0 -fno-inline -fno-omit-frame-pointer")
```

Comment:
This is going to force -g -O0 for all builds, even release which we don't want. We should let cmake use its automated handling to handle optimization flags.

Reply:
Sounds good; will remove :)